ITSchoolBankApp/Bank.cpp

```diff
@@ -274,7 +279,34 @@ void Bank::modificareCont()
 			std::cout << "Operatiune finalizata cu succes!\n";
 			Sleep(1500);
 			break;
+		case'3':
+			system("CLS");
+			float depunere;
+			std::cout << "Introduceti suma pe care doriti sa o depuneti in cont:\n";
+			std::cin >> depunere;
+			temp->setSoldValue(contCurent + depunere);
+			std::cout << "Operatiune finalizata cu succes!\n";
+			Sleep(1500);
```

Comment:
poate in loc de sleep ai putea sa folosesti ceva gen "apasati orice tasta pentru a merge mai departe". Sleep-ul ii fix ca timp si pune tot thread-ul pe pauza